ffmpeg3/plan.sh

```diff
@@ -0,0 +1,12 @@
+source "$(dirname "${BASH_SOURCE[0]}")/../ffmpeg/plan.sh"
+
+pkg_name=ffmpeg
+pkg_origin=core
+pkg_version=3.4.6
+pkg_source=https://ffmpeg.org/releases/${pkg_name}-${pkg_version}.tar.gz
+pkg_shasum=18f80cc9ca322134ed40d25d7489af954fa519b4e7e6289b7084f1b0a1cdf472
```

Comment:
Looks like the last commit that bumped the version didn't update the checksum.  Could you provide that and give this branch squash and then I think this is good to merge 😄